### PR TITLE
remove task by id

### PR DESCRIPTION
**Pull request by ywang282**
as above

### CODE REVIEW

File: controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Tasks.java [hunk 1]
```diff
@@ -82,4 +83,22 @@ public Response findTasks(@QueryParam("clusterName") Optional<String> clusterNam
     return Utils.buildResponse(HttpStatus.OK_200, result);
   }
 
+  /**
+   * Remove a task by its id
+   *
+   * @param id id of the task being removed
+   * @return Response object
+   */
+  @POST
```
Comment (angxu) [inline controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Tasks.java]: why not use DELETE?

File: controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Tasks.java [hunk 2]
```diff
@@ -82,4 +83,22 @@ public Response findTasks(@QueryParam("clusterName") Optional<String> clusterNam
...
+  @Path("/remove/{id : [0-9]+}")
```
Comment (angxu) [inline controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Tasks.java]: If you use DELETE, the path can be just "/{id: [0-9]+}".

Comment (shuzhang1989) [inline controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Tasks.java]: +1

File: controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Tasks.java [hunk 3]
```diff
@@ -27,6 +27,7 @@
 import java.util.List;
 import java.util.Optional;
 import javax.ws.rs.GET;
+import javax.ws.rs.DELETE;
```
Comment (shuzhang1989) [inline controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Tasks.java]: nit: alphabetical order

**Pull request merged**